content/security/advisory/2010-07-05.adoc

```diff
@@ -0,0 +1,30 @@
+---
+layout: simplepage
+title: Hudson Security Advisory 2010-07-05
+section: security
+kind: core
+---
+
+## Vulnerability
```

Comment:
For the record, the preferred asciidoc headers are with `==` instead of `##`.

:man_shrugging: